api/src/main/java/uk/gov/dhsc/htbhf/claimant/ApiDocumentation.java

```diff
@@ -23,13 +29,19 @@
 @Slf4j
 public class ApiDocumentation {
 
-    @Value("${app.version:}") // use APP_VERSION env variable if available, otherwise give no version info
+    @Value("${app.version:1.0}") // use APP_VERSION env variable if available, otherwise give no version info
```

Comment:
We need to update the comment: `// use APP_VERSION env variable if available, otherwise default to 1.0`

Reply:
The reason for this change is that without a version number, the swagger isn't valid